### ISSUE TITLE
[Android] User is navigated to the collectible detail page after collectible sending instead of the activity tab

### ISSUE THREAD

**Issue by VolodLytvynenko** [status-im/status-react#20701]
### Steps:
1. Open an account (this step is required; don't send collectibles from the main wallet page).
2. Open collectible -> tap 'send'
3. Confirm collectible sending

### Actual result:
After the collectible sending confirmation, the collectible detailed page is shown

https://github.com/status-im/status-mobile/assets/52490791/b00f8f9b-1fb7-4706-ba0c-f4c23d291910

### Expected result:
After the collectible sending confirmation, the activity tab is shown

### Devices:
- Pixel 7a, Android 13

### Logs
[Status-debug-logs (1).zip](https://github.com/user-attachments/files/16146964/Status-debug-logs.1.zip)

### ENV:
Nightly 9 Jul, 2024

**Comment by OmarBasem** [comment]
@VolodLytvynenko I am no longer reproducing this issue

**Comment by VolodLytvynenko** [comment]
> @VolodLytvynenko I am no longer reproducing this issue

@OmarBasem hey. Still reproducible on my side. Used this PR https://github.com/status-im/status-mobile/pull/20902


https://github.com/user-attachments/assets/bb20aebe-49d3-4669-8709-3ee384f32cce



### Logs:
[Status-debug-logs (1).zip](https://github.com/user-attachments/files/16426972/Status-debug-logs.1.zip)

**Comment by OmarBasem** [comment]
@VolodLytvynenko the current issue is about collectibles, right?

**Comment by VolodLytvynenko** [comment]
> @VolodLytvynenko the current issue is about collectibles, right?

@OmarBasem correct. This issue is related only to collectibles sending

**Comment by OmarBasem** [comment]
@VolodLytvynenko I see the video you posted is for assets not collectibles

**Comment by VolodLytvynenko** [comment]
> @VolodLytvynenko I see the video you posted is for assets not collectibles

@OmarBasem apologies. I mixed up the videos. I've just updated [here](https://github.com/status-im/status-mobile/issues/20701#issuecomment-2258127958)

**Comment by J-Son89** [comment]
@OmarBasem, @churik, @VolodLytvynenko - I think this [flow](https://github.com/status-im/status-mobile/issues/20701#issuecomment-2258127958) looks "good enough". 
Maybe we can just leave this as is and avoid adding any new bugs in hope of fixing this one with such a short timeframe? 
wdyt?

**Comment by OmarBasem** [comment]
@J-Son89 I agree, doesn't look like a big issue to me

**Comment by ulisesmac** [comment]
A side note on this issue:

We should rework/extend the wizard to abstract a more general behavior so that the code is easier to maintain, right now the logic to handle all flows is complex.

CC: @J-Son89  @mmilad75

**Comment by J-Son89** [comment]
Agreed @ulisesmac - I think we should also consider removing some of the entry points to make it easier to maintain and reduce the test coverage
